Bump worship-posts and worship-services consumers to v0.1.0

claire-lovisa opened the pull request:
# Context

**DL-6424**

The ticket has for goal to bump the worship mandatarissen consumer to its most recent version to keep the application up-to-date.

# Strategy

We can make two observations on this app:
- WOP is a read-only application, users can't edit anything or enter any data. All the data contained in this application is coming from either migrations or consumers
- there are two consumers that would need a bump: `worship-services-sensitive-consumer` as requested in the Jira ticket but also `worship-posts-consumer`

From this, I decided to:
1. Bump both consumers at the same time
2. Flush the database entirely instead of writing flushing migrations to ease up the bumping process

# Notes

The `worship-posts-consumer` doesn't need the remapping per say, it could completely work without. When developping though, I noticed that with remapping enabled, the initial sync was significantly faster. This is why I kept it, and the mapping rule is just to pass everything to the ingest graph.

# How to test

Please refer to the Changelog for instructions on how to test this, in case it would lack of something don't hesitate to mention it so that I can improve the Changelog!

claire-lovisa committed:
Bump worship-posts and worship-services consumers

claire-lovisa committed:
Fix readme and changelog

benjay10 commented on CHANGELOG.md:
````diff
@@ -0,0 +1,67 @@
+```
+  worship-services-sensitive-consumer:
+    environment:
+      DCR_SYNC_BASE_URL: 'https://loket.lblod.info/' # The endpoint of your choice
````
This seems to be the URL for Loket QA? Is that what we want in the deploy notes? The one below this line is the same.

claire-lovisa replied:
Mmh it doesn't matter too much I think, but as the most critical deploy is prod, i'll put prod's URI to be sure!

benjay10 reviewed (commented):
I don't see any obvious problems. I don't have a database, nor the backup scripts on my local machine to test the backup steps, but with the correct PROD URL of Loket, data seems to be consuming.

claire-lovisa committed:
Update changelog and readme

claire-lovisa committed:
Remove phone cleanup mapping rule

WOP behaves differently than OP for phone numbers: here we just ingest
them as they are in Loket, we don't clean them upfront

claire-lovisa committed:
Remove reasoner service, replaced by new consumer logic

claire-lovisa committed:
Allow phone numbers to be consumed directly

claire-lovisa committed:
Bump frontend

mirdono reviewed (approved):
Works as expected now, when combined with [#3](https://github.com/lblod/worship-positions-graph-dispatcher-service/pull/3) the expected data changes come through.

claire-lovisa committed:
Bump positions-dispatcher
